tests/src/test/java/alluxio/client/fs/FileOutStreamIntegrationTest.java

```diff
@@ -122,46 +124,45 @@ public void writeTwoByteArrays() throws Exception {
    * Tests writing to a file and specify the location to be localhost.
    */
   @Test
-  @LocalAlluxioClusterResource.Config(confParams = {
-      PropertyKey.Name.USER_BLOCK_WRITE_LOCATION_POLICY,
-      "alluxio.client.block.policy.LocalFirstPolicy"
-      })
   public void writeSpecifyLocal() throws Exception {
+    ServerConfiguration.set(PropertyKey.USER_BLOCK_WRITE_LOCATION_POLICY,
+        "alluxio.client.block.policy.LocalFirstPolicy");
+
     AlluxioURI filePath = new AlluxioURI(PathUtils.uniqPath());
     final int length = 2;
     CreateFilePOptions op = CreateFilePOptions.newBuilder().setWriteType(mWriteType.toProto())
         .setRecursive(true).build();
-    try (FileOutStream os = mFileSystem.createFile(filePath, op)) {
+    try (FileOutStream os = sFileSystem.createFile(filePath, op)) {
       os.write((byte) 0);
       os.write((byte) 1);
     }
     if (mWriteType.getAlluxioStorageType().isStore()) {
-      checkFileInAlluxio(filePath, length);
+      FileOutStreamTestUtils.checkFileInAlluxio(sFileSystem, filePath, length);
     }
     if (mWriteType.getUnderStorageType().isSyncPersist()) {
-      checkFileInUnderStorage(filePath, length);
+      FileOutStreamTestUtils.checkFileInUnderStorage(sFileSystem, filePath, length);
     }
   }
 
   /**
-   * Tests writing to a file for longer than HEARTBEAT_INTERVAL_MS to make sure the sessionId
-   * doesn't change. Tracks [ALLUXIO-171].
+   * Tests writing to a file for longer than {@link PropertyKey#WORKER_BLOCK_HEARTBEAT_INTERVAL_MS}
+   * to make sure the sessionId doesn't change. Tracks [ALLUXIO-171].
    */
   @Test
   public void longWrite() throws Exception {
     AlluxioURI filePath = new AlluxioURI(PathUtils.uniqPath());
     final int length = 2;
-    try (FileOutStream os = mFileSystem.createFile(filePath, CreateFilePOptions.newBuilder()
+    try (FileOutStream os = sFileSystem.createFile(filePath, CreateFilePOptions.newBuilder()
         .setWriteType(mWriteType.toProto()).setRecursive(true).build())) {
       os.write((byte) 0);
-      Thread.sleep(Constants.SECOND_MS * 2);
+      Thread.sleep(ServerConfiguration.getMs(PropertyKey.WORKER_BLOCK_HEARTBEAT_INTERVAL_MS));
```

Comment:
does this have to have a x2 ?

Reply:
Not sure. I can put it back to be safe? 

The original idea around this test was to make sure that the session id doesn't change.

Reply:
added the *2 back